content/W2015/W15-event-grad-info-session.md

```diff
@@ -0,0 +1,16 @@
+Title: Graduate info Session
+Date: 2015-02-04 05:30pm
```

Comment:
I don't know if the Pelican engine can parse times written in this format. To be consistent, we should be using 24h time.